uint8  motor_id
string item_name
---
bool   success
uint32 value
